shell/platform/android/android_context.cc

```diff
@@ -0,0 +1,45 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/android/android_context.h"
+
+#include "flutter/shell/platform/android/android_context_gl.h"
+#include "flutter/shell/platform/android/android_environment_gl.h"
+
+namespace flutter {
+
+AndroidContext::AndroidContext(AndroidRenderingAPI rendering_api)
+    : rendering_api_(rendering_api) {}
+
+AndroidContext::~AndroidContext() = default;
+
+AndroidRenderingAPI AndroidContext::RenderingApi() {
+  return rendering_api_;
+}
+
+bool AndroidContext::IsValid() {
+  return true;
```

Comment:
What is the point of this method if it is always true? It seems you are not returning a non-valid context from the factory. If you don't plan on adding to this, I'd get rid of this.

Reply:
Done. This has been removed.